lochness/email/__init__.py

```diff
@@ -104,7 +108,8 @@ def send_detail_google(Lochness,
     s.quit()
 
 
-def send_out_daily_updates(Lochness, days: int = 1, test: bool = False):
+def send_out_daily_updates(Lochness, days: int = 1,
+                           test: bool = False, mailx: bool = True):
```

Comment:
Okay I understand the use of `mailx` bool but where did we use this new function? I see one use in `test_email.py` but none in the main code.

Reply:
it's under `scripts/sync.py`

Reply:
Okay then why is there no update in `scripts/sync.py` in this PR?